internal/controller/controllers/hypershiftagentserviceconfig_controller.go

```diff
@@ -189,8 +189,18 @@ func (hr *HypershiftAgentServiceConfigReconciler) Reconcile(origCtx context.Cont
 		return ctrl.Result{}, nil
 	}
 
+	supportsCertManager := false
+	if !asc.rec.IsOpenShift {
+		var certErr error
+		supportsCertManager, certErr = serverSupportsCertManager(ctx, asc.Client)
+		if err != nil {
```

Comment:
```suggestion
		if certErr != nil {
```
Should this check for `certErr`?